packages/beeq/src/components/panel/bq-panel.tsx

```diff
@@ -138,10 +138,12 @@ export class BqPanel {
   render() {
     return (
       <div
-        class="bq-panel"
+        class={{
+          'bq-panel': true,
+          'panel-hidden': !this.open,
+        }}
```

Comment:
This affects everything that uses the bq-panel, e.g. the dropdown, and the select, where the panel in hidden mode (not open) seems to visually fade out when the component renders, normally the panel shouldn't be visible.

https://github.com/Endava/BEEQ/assets/328492/cdce9321-1f5d-48c4-9b64-33a50c86e303